Task1/CommaSeperator.js

```diff
@@ -0,0 +1,5 @@
+// 1- https://www.codewars.com/kata/56e2f59fb2ed128081001328/train/javascript
+function printArray(array){
+    return array.join(",");
```

Comment:
Array method join, by default take comma(,) as the separator.
No need to specify a separator if you want to separate them by comma.

array.join();

Look into the document link below to learn more about array.join():
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/join

They is nothing wrong in specifying the separator comma.